en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md

```diff
@@ -24,7 +24,7 @@ This section explains how to attach a custom workflow to the user signup operati
 #### Configuring the Enterprise Integrator
 
 !!! note
-    Follow this sub section, only if you will be using the **Entreprise Integrator(EI)** for the business process tasks. If not please refer the sub section for [Configuring the Business Process Server.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-business-process-server)
+    Follow this sub section, only if you will be using the **Enterprise Integrator(EI)** for the business process tasks. If not please refer the sub section for [Configuring the Business Process Server.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-business-process-server)
```

Comment:
This is not part of this exercise, but this link will appear in a page not found as there is no {{base_path}} in front of the path.

Reply:
Fixed.